kratos/processes/compute_nodal_gradient_process.cpp

```diff
@@ -224,8 +261,15 @@ ComputeNodalGradientProcess<3, component_type, NonHistorical>::ComputeNodalGradi
     KRATOS_TRY
     
     VariableUtils().CheckVariableExists(rOriginVariable, mrModelPart.Nodes());
-    KRATOS_ERROR_IF_NOT(rModelPart.Nodes().begin()->Has( rGradientVariable )) << "Missing variable " << rGradientVariable << std::endl;
```

Comment:
shouldn't this still be checked depending on the template-arg?

Reply:
The area variable is always non-historical